web/v2/index.ts

```diff
@@ -185,7 +185,7 @@ export const APIWebRoutes = {
    *
    * @returns APIResponse<APIPlayer>
    */
-  player(id: bigint) {
+  player(id: bigint | number | string) {
```

Comment:
`bigint` should already cover `number`, I believe.

Also, I am not really a fan of allowing string in all these requests that require an identifier.

Reply:
Removed string where numeric identifiers are expected. As for bigint/number, [neither is compatible with the other](https://www.typescriptlang.org/play/#code/GYVwdgxgLglg9mABABwE4zFAQjA5hqACgA8AuRAIzwIEpyA3OGAEwCgBvVxbxVAUyghUSCAgDOcADZ8AdJLi4SNANysAvq1ahIsBCnSYAciAC2FPqhLkwp86jqJGLDlx79BwxKLATpchUqqGqxoBDj4mIQAjCohBlDGZhbRYCpAA)

Reply:
Good to know. Thank you for providing the example, too!